notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-756_analysis.ipynb

```diff
@@ -0,0 +1,701 @@
+{
```

Comment:
<p>As already commented, I think that it would be better to keep the output and plots in a sub-directory of the notebook directory. It would improve the re-usability of the notebook by someone else.</p><p> </p><p>The notebook looks very good. It would be nice to add a final comment cell with the conclusions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>